src/FakeItEasy.Analyzer/FakeItEasy.Analyzer.targets

```diff
@@ -0,0 +1,10 @@
+<Project>
+
+    <Target Name="WarnAboutWrongAnalyzerPackage" AfterTargets="AfterBuild">
+        <Warning Text="The FakeItEasy.Analyzer package is no longer maintained and does nothing. Use the C# specific analyzer package instead: FakeItEasy.Analyzer.CSharp."
+                 Condition=" '$(Language)' == 'C#' " />
+        <Warning Text="The FakeItEasy.Analyzer package is no longer maintained and does nothing. Use the VB.NET specific analyzer package instead: FakeItEasy.Analyzer.VisualBasic."
+                 Condition=" '$(Language)' == 'VB' " />

```

Comment:
I'm not 100% sure about this. I'm not certain the `Language` property is always defined. Maybe there should be a generic message in case we can't determine the project language.

Reply:
So leave the two that are here and add a third with no condition? Sounds nice and safe.

Reply:
Well, I still need a condition, otherwise there will be two warnings when `Language` is defined... Or maybe I could use [`Choose`](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditional-constructs?view=vs-2017).

Reply:
`Choose` doesn't seem to work in a `Target`...

Reply:
I'd wondered. It may be possible to phrase things so the two warnings complement one another. For example,

"The FakeItEasy.Analyzer package is no longer maintained and does nothing. Language-specific analyzer packages are provided instead." could always be emitted.

And then one of 

"Use the VB.NET-specific analyzer package instead: FakeItEasy.Analyzer.VisualBasic."

or 

"Use the C#-specific analyzer package instead: FakeItEasy.Analyzer.CSharp."

if the language condition is met.

If the warning order is preserved, it's not terrible, I think.